src/jit/lsra.cpp

```diff
@@ -3475,12 +3475,12 @@ LinearScan::updateRegStateForArg(LclVarDsc* argDsc)
 #endif
             );
 
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
This #ifdef is not needed since lvIsHfaRegArg direct field reference is replaced with lvIsHfaRegArg() accessor.